plugins/memcached.yaml

```diff
@@ -0,0 +1,33 @@
+# Plugin Info
+version: 0.0.1
+title: Memcached
+description: Log parser for Memcached
+parameters:
+  file_path:
```

Comment:
Should plugins that have a single file input be log_path to match format of other plugins that have multiple log paths use something like general_log_path?

Reply:
These parameter names were pulled directly from the log bundles when I generated them. I know Zack put a lot of effort into standardizing the names across bundles, so I'm guessing there's probably a good reason we did file_path over log_path. My guess is that file_path was used when there is typically one file to point at, whereas log_path was used when you can't point at a directory or glob pattern.